plugin-management-library/pom.xml

```diff
@@ -50,6 +50,11 @@
             <version>1.19.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
should this be removed from the CLI now? https://github.com/jenkinsci/plugin-installation-manager-tool/blob/master/plugin-management-cli/pom.xml#L34